File: Java/number.java
public class number {
    public static void main(String[] args) {
        int n = 5;
        int count = 0;
        for (int i = 1; i < n; i++) {
            for (int j = 0; j < n-i; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j < i; j++) {
                count = count + 1;
                System.out.print(count + " ");
            }
            System.out.println();
        }
    }
}

File: java_to_js/number.js
function number(n){
    let count = 0;
    let output = "";
    for(let i = 1; i < n; i++){
        let row = "";
        for(let j = 0; j < n - i; j++){
            row += " ";
        }
        for(let j = 0; j < i; j++){
            count = count + 1;
            row += count + " ";
        }
        output += row + "<br>";
    }

    return output;
}